frame/nomination-pools/src/lib.rs

```diff
@@ -2144,8 +2144,8 @@ pub mod pallet {
 impl<T: Config> Pallet<T> {
 	/// Returns the pending rewards for the specified `member_account`.
 	///
-	/// In the case of error, `None` is returned.
-	pub fn pending_rewards(member_account: T::AccountId) -> Option<BalanceOf<T>> {
+	/// In the case of error, `None` is returned. Used by runtime API.
+	pub fn api_pending_rewards(member_account: T::AccountId) -> Option<BalanceOf<T>> {
```

Comment:
I saw this function highlighted and the double `if let` below it, and felt a re-write was in order - only then to realise that it was not part of this PR. 

In any case, more idiomatic function below if you choose to refresh it alongside the function name.

```
    /// Returns the pending rewards for the specified account.
	///
	/// In the case of error, `None` is returned. Used by runtime API.
	pub fn api_pending_rewards(who: T::AccountId) -> Option<BalanceOf<T>> {
		match PoolMembers::<T>::get(who) {
			None => None,
			Some(pool_member) => 
				RewardPools::<T>::get(pool_member.pool_id)
					.zip(BondedPools::<T>::get(pool_member.pool_id))
					.map(|(reward_pool, bonded_pool)| {
						let current_reward_counter = reward_pool
							.current_reward_counter(pool_member.pool_id, bonded_pool.points)
							.ok()?;
						pool_member.pending_rewards(current_reward_counter).ok()
					}).flatten()
		}
	}
```

Reply:
Both look fine to me 👍

Reply:
I'm going to resolve this thread since, as Ross mentioned, the `api_pending_rewards` implementation is not in the context of this PR and the current impl is OK as per Kian.